services/app-api/package.json

```diff
@@ -22,20 +22,22 @@
     "@types/jsdom": "^20.0.0",
     "aws-lambda": "^1.0.7",
     "jest": "^27.4.7",
-    "ksuid": "^3.0.0",
```

Comment:
Comment: Ah, this almost tripped me up but I see we just had this under both `devDependencies` and `dependencies` and only need it under normal deps. 👍

Reply:
yeah, it was a non-issue under node 14, but under 18 devDep takes priority over reg dep, causing ksuid to be missing at runtime